index.html

```diff
@@ -59,10 +62,75 @@
     </div>
   </nav>
 
+  <!--Our Customers-->
+  <div class="slider-1">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <h4>
+            Trusted by over <span class="blue">1500</span> customers worldwide
+          </h4>
+          <hr class="section-divider" />
+
+          <div class="slider-container">
+            <div class="swiper-container image-slider">
+              <div class="swiper-wrapper">
+                <div class="swiper-slide">
+                  <img
+                          class="img-fluid"
+                          src="./public/assets/images/customer-logo-1.png"
+                          alt="alternative"
```

Comment:
Forgot to tell you to set the `alt` attributes properly (they should include a description of the picture).

Reply:
This applies to all `<img>` tags you add.